spoken/settings.py

```diff
@@ -141,7 +145,7 @@
         'USER': MDB_USER,
         'PASSWORD': MDB_PASS,
         'HOST': MDB_HOST,                  # Empty for localhost through domain sockets or '127.0.0.1' for localhost through TCP.
-        'PORT': '',                  # Set to empty string for default.
+        'PORT': MDB_PORT,                  # Set to empty string for default.
```

Comment:
for this as well

Reply:
ok. What about 'HOST': MDB_HOST, ?

Reply:
You can do the same for all of them. You can check or copy them from settings file in the main repo.